lesson15/main.js

```diff
@@ -4,8 +4,10 @@ const closeButton = document.getElementById("js-close-button");
 const overlay = document.getElementById("js-overlay");
 const openListButton = document.getElementById("js-open-list-button")
 const ul = document.getElementById("js-ul");
-const numberInput =document.getElementById("js-number-input")
-const validationMessage = document.getElementById("js-validation");
+const form = document.getElementById("js-form");
+const errorMessages = document.getElementsByClassName("js-error-message");
+const userNumberInput = document.getElementById("js-number-input");
+const userNameInput = document.getElementById("js-name-input");
```

Comment:
IMO
Do you use 'const openListButton'?

MUST
For 'const userNumberInput' and 'const userNameInput', since they are declared twice, one of them should be removed.

Reply:
Thank you for the comment. 
I overlooked the duplicated variables when modifying the eventListener and validation function.
Deleted the dupes, please check the commit
6ad544c96b5c4e21ce5ff7a81fa84174f72e7142